src/components/user-steps-wrapper/UserStepsWrapper.tsx

```diff
@@ -34,12 +33,12 @@ const UserStepsWrapper: FC<UserStepsWrapperProps> = ({ userRole }) => {
       key='1'
       setIsUserFetched={setIsUserFetched}
     />,
-    <SubjectsStep key='2' />,
```

Comment:
I suppose we shouldn't delete this, but use conditional rendering instead. Because UserSpepsWrapper can be used by tutor-user aswell and he'll need this to be rendered.